src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+                                <MobileHamburgerCrossBox onClick={toggle}>
+                                    <MobileHamburgerCross>
+                                        <MobileHamburgerCross1 />
+                                    </MobileHamburgerCross>
+                                </MobileHamburgerCrossBox>
+                                <NavLink to="home" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenuUp>Главная</MobileHamburgerSectionMenuUp>
+                                </NavLink>
+                                <NavLink to="planning" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenu>Планирование</MobileHamburgerSectionMenu>
+                                </NavLink>
+                                <NavLink to="awards" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenu>Награды</MobileHamburgerSectionMenu>
+                                </NavLink>
+                                <NavLink to="contacts" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenu>Контакты</MobileHamburgerSectionMenu>
+                                </NavLink>
+                            </MobileHamburgerbackground>
+                        </>
+                    }
+                    {width <= 529 && <>                 {/* --------------- Mobile ------------- */}
+                        <ContainerHeder>
+                            <LeftBlock>
+                                <LogoHead />
+                                <PointBalUser />
+                            </LeftBlock>
+                            <MobileHamburgerContainer onClick={toggle} >
+                                <MobileHamburgerLine />
+                            </MobileHamburgerContainer>
+                        </ContainerHeder>
+                        <MobileHamburgerbackground style={{ transform: isOpen ? "translateX(0%)" : "translateX(120%)", opacity: isOpen ? "1" : "0.3" }}>
+                            <MobileHamburgerLoginBox>
+                                <MenuHeaderBlock>
+                                    <MenuInitiaLetterCircle>
+                                        <MenuInitiaLetter>{nameLetter}</MenuInitiaLetter>
+                                    </MenuInitiaLetterCircle>
+                                    <MenuName>{UserName}</MenuName>
+                                    <NavLink to="login" onClick={`${toggle}${logOut}`}>
```

Comment:
Its some hack to add 2 event handlers?

Reply:
I came up with this idea, I tried it, and it works. I don't know if it's the right thing to do. If it is permissible, then let it be a feature of our small team. If not, I'll fix it somehow and write differently))